packages/intl-enumerator/data/currencies.ts

```diff
@@ -0,0 +1,32 @@
+export type Currency = typeof currencies[number];
+
+export const currencies = [
+  'AED', 'AFN', 'ALL', 'AMD', 'ANG', 'AOA', 'ARS', 'AUD', 'AWG', 'AZN', 'BAM',
```

Comment:
do these come from [cldr-json?](https://github.com/unicode-org/cldr-json) if yes we do have prior arts on pre-processing those dataset in other polyfills you can check out

Reply:
@longlho currencies are coming from this file: https://raw.githubusercontent.com/unicode-org/cldr/master/common/validity/currency.xml, but I can definitely substitute it with cld-json 🙂

Reply:
We should have some examples of pulling currencies from the numberformat polyfill